### ISSUE TITLE
Use form builder to create/remove form sections and fields for enquiries

### ISSUE THREAD

**Issue by baziwane** [rapidftr/tracker#55]
This feature shall enable any user with permissions to define form sections and fields for enquiries.

**Comment by baziwane** [comment]
Refactoring & Generifying Form Sections:

#### UI
- Make all views related to form sections (rendering fields, rendering form sections, etc) generic, i.e. make it shared
- Create separate Admin page for Enquiries
- _Highlighted fields for Enquiries_

#### Ruby
- Make FormSection and Field generic to both Children & Enquiry

**Comment by rdsubhas** [comment]
Hi @ctumwebaze @andreweskeclarke Getting an error when migrating old data (not when actually doing the migration, but when running the app). Temporarily cleaned up data in dev.rapidftr.com.

To reproduce: `git checkout` to some commit before these changes, do an app:reset, then checkout to master, migrate and run the application. It will crash in the Children view page.

TL;DR - Old data not getting migrated.

**Comment by tomclement** [comment]
@ctumwebaze @andreweskeclarke - I experienced the same issue as Subhas. Also, the _Highlighted fields for Enquiries_ task does not seem to have been covered. @baziwane are we planning to track this separately (currently the only other unplayed highlighting story - #143 seems to cover separation for web and mobile only)

**Comment by tomclement** [comment]
The _Highlighted fields for Enquiries_ task is covered in #149 - so no need to tag it to this card. All issues related to upgrades and deployment to servers with existing data will now be tracked in #153. This includes the error discovered by @rdsubhas in https://github.com/rapidftr/tracker/issues/55#issuecomment-50636994
